.github/workflows/test.yml

```diff
@@ -40,8 +56,20 @@ jobs:
         with:
           shared-key: release-${{ matrix.target }}
       - run: cargo test --all-features
-        env:
-          CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: aarch64-linux-gnu-gcc
       - run: cargo fmt --check
-        env:
-          CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: aarch64-linux-gnu-gcc
+
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Cache
+        uses: Swatinem/rust-cache@v2
+        # Ensure that docs can also build
+      - run: cargo doc --all-features --no-deps
+
+  results:
+    runs-on: ubuntu-latest
+    needs: [build, docs]
+    steps:
+      - name: results
+        run: echo '${{ toJSON(needs.*.result) }}'
```

Comment:
Does this actually fail a build?

Reply:
If one of the prior "needs" steps fail, then this check will never run. So wont report, and wont allow a check to pass a PR.